pyi.py

```diff
@@ -28,6 +28,8 @@
 
 if sys.version_info >= (3, 9):
     from ast import unparse
+
+
```

Comment:
Huh, I was expecting these random extra two lines I added to be auto-reformatted by black.